Makefile

```diff
@@ -20,10 +20,10 @@ KAKAROT_COMMIT := .katana/remote_kakarot_sha
 .PHONY: $(EF_TESTS_DIR)
 setup: $(EF_TESTS_DIR)
 
-fetch-dump:
+fetch-dump: fetch-kakarot-submodule-commit
```

Comment:
this was changed because you had issues with loading the latest commit's dump, oui?

Reply:
yes, thank you
problem was that if the fetch already existed, it wouldnt fetch once more